### PR TITLE
Add script/travis so all people can play with Travis-CI images.

### PR DESCRIPTION
**Pull request by envygeeks**

### CODE REVIEW

File: script/travis [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
```
Comment (parkr) [inline script/travis]: Would you mind adding usage info up here?

Comment (envygeeks) [inline script/travis]: I'm gonna be honest, this is for programmers so if they can't get usage info from the simple 2 if's they shouldn't be using it.  I'm on the fence about whether I wish to comply with this because the script is so simple.

Comment (envygeeks) [inline script/travis]: Let me rephrase that, did you understand how to use it @parkr?

Comment (parkr) [inline script/travis]: I think it's simple enough, but usage examples save time. It took me a second to understand what $1 and $2 were supposed to do because I had to hunt. I sometimes have to get usage help for things because I use so many tools every day that I can mix them up sometimes.

Comment (envygeeks) [inline script/travis]: That's fair then, after I finish up the ObjectSpace removal I'll add a usage info.

**Comment by envygeeks** [comment]
Done.

**Comment by parkr** [comment]
You rock! :tada:

**Comment by parkr** [comment]
@jekyllbot: merge +dev
